packages/svelteui-core/src/lib/components/Title/Title.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+	import { Text } from '$lib';
```

Comment:
When importing a component into another component, I would do `../` instead of $lib because they are pretty close to each other